public/js/upload.js

```diff
@@ -0,0 +1,16 @@
+document.addEventListener("DOMContentLoaded", init, false);
+
+function init() {
+    document.querySelector('#fileChooser').addEventListener('change', handleFileSelect, false);
+}
+
+function handleFileSelect(event) {
+    if(!event.target.files) return;
+    const files = event.target.files;
+    const selected = document.querySelector("#selectedFiles ul");
+    selected.innerHTML = "";
+    for(var i=0; i<files.length; i++) {
```

Comment:
If we linted, it would be going crazy :stuck_out_tongue_winking_eye: 

Could also be `files.forEach`.